modules/extensions-support/src/main/resources/META-INF/services/org/mule/runtime/core/config/registry-bootstrap.properties

```diff
@@ -0,0 +1,2 @@
+extensions.singletx.transaction.factory1=org.mule.runtime.module.extension.internal.runtime.transaction.ExtensionTransactionFactory
+extensions.singletx.transaction.resource1=org.mule.runtime.module.extension.internal.runtime.transaction.ExtensionTransactionKey
```

Comment:
Is strange to me that the transaction resource is ExtensionTransactionKey instead of ExtensionTransactionResource